typing/env.mli

```diff
@@ -127,8 +127,13 @@ val lookup_cltype:
   ?loc:Location.t -> ?mark:bool ->
   Longident.t -> t -> Path.t * class_type_declaration
 
-val copy_types: string list -> t -> t
-  (* Used only in Typecore.duplicate_ident_types. *)
+type copy_of_types
+
```

Comment:
Nitpicking: since those 3 definitions are a set, no need to have empty lines between them.